src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -46,13 +51,17 @@ public class Drivetrain extends SubsystemBase {
   private SlewRateLimiter m_rotLimiter;
 
   private Timer m_timer;
-  private double m_prevTime;
+  private double m_prevSlewRateTime;
 
   // Odometry class for tracking robot pose
   SwerveDriveOdometry m_odometry;
+  private Pose2d m_pose;
+  private ChassisSpeeds m_relativeSpeeds;
 
-  /** constructs a new Drivatrain object */
-  public Drivetrain() {
+  private SwerveModulePosition[] m_swerveModulePositions;
+
+  /** constructs a new Drivetrain object */
+  public Drivetrain(Vision vision) {
```

Comment:
Why are we consuming Vision? It doesn't even appear to be used...?

Reply:
It's required to be able to update the swerve drive pose estimator with it, once it's properly implemented.

Reply:
If it's not used here then it should not be included here. Vision can put it in their branch if necessary.

Reply:
I've removed the argument now